src/lookup-input.js

```diff
@@ -3,8 +3,14 @@ import React, {PropTypes as t} from 'react';
 type LookupInputPropsType = {
   img: string,
   onChange?: () => any,
-  onClick?: () => any,
-  onKeyPress?: () => any
+  onClick?: () => any
+};
+
+type EventType = {
+  target: {
+    name: string,
```

Comment:
It looks like you aren't using the `name` property of the event, but you are using `which`.